tests/plotting/test_plotting.py

```diff
@@ -26,6 +25,17 @@
 from pyvista.plotting.plotting import SUPPORTED_FORMATS
 from pyvista.utilities.misc import can_create_mpl_figure
 
+# Image regression warning/error thresholds for releases after 9.0.1
+# TODO: once we have a stable release for VTK, remove these.
+
+# these images vary between Windows when using OSMesa and Linux/MacOS
+# and will not be verified
+
+
+# these images vary between Linux/Windows and MacOS
+# and will not be verified for MacOS
+
+
```

Comment:
These comments seem to be leftovers from the old design, and are somewhat confusing here.